.env.example

```
@@ -20,6 +20,12 @@ JWT_SECRET=<your-jwt-secret-at-least-32-bytes-long>
 CORS_ALLOWED_ORIGINS=http://localhost:3000,http://localhost:5173,http://localhost:5174,http://localhost:5175,http://localhost:5176
 AIRFLOW_UID=50000
 AIRFLOW_WEBHOOK_SECRET=change-me-airflow-webhook-secret
+AIRFLOW_API_BASE_URL=http://airflow-apiserver:8080
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The AIRFLOW_API_BASE_URL key should go before the AIRFLOW_UID key

(UnorderedKey)